tools/spanner/purge_ttl.py

```diff
@@ -41,39 +42,103 @@ def from_env():
     return (instance_id, database_id)
 
 
-def spanner_read_data(request=None):
-    (instance_id, database_id) = from_env()
-    instance = client.instance(instance_id)
-    database = instance.database(database_id)
+def deleter(database, name, query):
+    with statsd.timer("syncstorage.purge_ttl.{}_duration".format(name)):
+        logging.info("Running: {}".format(query))
+        start = datetime.now()
+        result = database.execute_partitioned_dml(query)
+        end = datetime.now()
+        logging.info("{}: removed {} rows, batches_duration: {}".format(
+            name, result, end - start))
+
+
+def add_conditions(args, query):
+    if args.collection_ids:
+        query += " AND collection_id"
+        if len(args.collection_ids) == 1:
+            query += " = {:d}".format(args.collection_ids[0])
+        else:
+            query += " in ({})".format(
+                ', '.join(map(str, args.collection_ids)))
```

Comment:
I always prefer bind params :) but I guess this'll do for now

Reply:
Heh, so do I, but it would have made things really complicated. Hopefully we trust @erkolson not to insert ";drop tables;"